geonode_mapstore_client/templatetags/get_menu_json.py

```diff
@@ -78,8 +73,6 @@ def get_base_right_topbar_menu(context):
     if is_mobile:
         return []
 
-    home = {"type": "link", "href": "/", "label": "Home"}
```

Comment:
@dsuren1 you have removed `home` but you didn't remove it in https://github.com/dsuren1/geonode-mapstore-client/blob/%231769_catalog_pages/geonode_mapstore_client/templatetags/get_menu_json.py#L108

It throws an error.

Reply:
Updated